wasm/wat/lexer.go

```diff
@@ -28,12 +28,12 @@ func lex(source []byte, parser parseToken) error {
 
 	// Web assembly expressions are grouped by parenthesis, even the minimal example "(module)". We track nesting level
 	// to help report problems instead of bubbling to the parser layer.
-	nestLevel := 0
+	parenDepth := 0
```

Comment:
wasm-tools uses the word "depth" which I think is better than "level" so borrowing that, too